server/api/documents.js

```diff
@@ -17,7 +17,7 @@ router.get(
       getDocumentFunction = docsDb.getUserDocuments;
     }
 
-    getDocumentFunction()
+    getDocumentFunction(id)
```

Comment:
Passing the id all the time here is a bit confusing as it should be passed only when we want to load the user's documents, not when loading documents for interpreters (even if it's not going to break anything as it will be ignored). Instead, you could set the id in line 17:
```
getDocumentFunction = () => docsDb.getUserDocuments(id);
```

Reply:
Done